platformio.ini

```diff
@@ -805,3 +806,21 @@ lib_deps =
   ${esp32.lib_deps}
   TFT_eSPI @ ^2.3.70
 board_build.partitions = ${esp32.default_partitions}
+
+# ------------------------------------------------------------------------------
+# Arduino® Nano ESP32
+# ------------------------------------------------------------------------------
+[env:arduino_nano_esp32]
+platform = https://github.com/platformio/platform-espressif32.git
+board = arduino_nano_esp32
+upload_protocol = esptool
+board_build.partitions = tools/WLED_ESP32_16MB.csv
+board_build.flash_mode = qio
+build_unflags = ${common.build_unflags}
+build_flags = ${common.build_flags} ${esp32s3.build_flags}
+  -D CONFIG_LITTLEFS_FOR_IDF_3_2 -D WLED_WATCHDOG_TIMEOUT=0
```

Comment:
Are you sure this one is optimal for latest core? Arduino core in platform 4 and above have LittleFS included (IDF v4).

Reply:
not 100% sure its needed - build-in LittleFS is always used, this flag is also there in other -S3 envs - maybe could be omitted.

Reply:
I've removed `-D CONFIG_LITTLEFS_FOR_IDF_3_2`.